### PR TITLE
OMIM ingest - March 2024

### PR DESCRIPTION
**Pull request by sabrinatoro**
Spreadsheet is [here](https://docs.google.com/spreadsheets/d/1pvDCV7wsw52oTtJ1VXop0n1Pvbma7-M20DCLm3rjztY/edit#gid=0)

### CODE REVIEW

**Comment by sabrinatoro** [comment]
Note that 
Bethlem myopathy 1 and Ullrich congenital muscular dystrophy 1 were both renamed ~1a in OMIM
I updated these label in Mondo because they both had only OMIM as an equivalent. 
Note that Bethlem myopathy 1 also has a UMLS equivalent. I will contact Megan and make appropriate changes (if needed) before the release. Please do not let this block this PR

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -198459,14 +198459,14 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0009681
-name: Ullrich congenital muscular dystrophy 1
+name: Ullrich congenital muscular dystrophy 1a
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:198462]: 1A should be capital:
https://omim.org/entry/254090

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -468159,11 +468161,11 @@ is_a: MONDO:0008004 {source="MONDO:Redundant", source="OMIM:157700"} ! familial
 
 [Term]
 id: MONDO:0024530
-name: Bethlem myopathy 1
+name: Bethlem myopathy 1a
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:468164]: 1A should be capital here too: https://omim.org/entry/158810

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -526995,6 +526998,137 @@ name: spermatogenic failure 89
...
+[Term]
+id: MONDO:0958232
+name: hearing loss, autosomal dominant 90
+synonym: "deafness, autosomal dominant 90" NARROW [OMIM:620722]
+xref: OMIM:620722 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019587 {source="OMIM:620722", source="https://orcid.org/0000-0002-4142-7153"} ! autosomal dominant nonsyndromic hearing loss
+
+[Term]
+id: MONDO:0958233
+name: Bethlem myopathy 1b
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:527053]: 1B should be capital: https://omim.org/entry/620725

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -526995,6 +526998,137 @@ name: spermatogenic failure 89
...
+xref: OMIM:620725 {source="MONDO:equivalentTo"}
+is_a: MONDO:0008029 {source="OMIM:620725", source="https://orcid.org/0000-0002-4142-7153"} ! Bethlem myopathy
+
+[Term]
+id: MONDO:0958234
+name: Bethlem myopathy 1c
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:527059]: IC should be capital

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -526995,6 +526998,137 @@ name: spermatogenic failure 89
...
+xref: OMIM:620726 {source="MONDO:equivalentTo"}
+is_a: MONDO:0008029 {source="OMIM:620726", source="https://orcid.org/0000-0002-4142-7153"} ! Bethlem myopathy
+
+[Term]
+id: MONDO:0958235
+name: Ullrich congenital muscular dystrophy 1b
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:527065]: 1B should be capital

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -526995,6 +526998,137 @@ name: spermatogenic failure 89
...
+xref: OMIM:620727 {source="MONDO:equivalentTo"}
+is_a: MONDO:0000355 {source="OMIM:620727", source="https://orcid.org/0000-0002-4142-7153"} ! Ullrich congenital muscular dystrophy
+
+[Term]
+id: MONDO:0958236
+name: Ullrich congenital muscular dystrophy 1c
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:527071]: 1C should be capital

**Review by nicolevasilevsky — APPROVED** [review]
minor typos, otherwise looks good

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:198462]: done

File: src/ontology/mondo-edit.obo (hunk 6 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:527071]: done

**Pull request merged**